pkgs/tools/graphics/shotgun/default.nix

```
@@ -0,0 +1,27 @@
+{ pkgs, stdenv, fetchFromGitHub, rustPlatform, ... }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "shotgun";
+  version = "2.2.0";
+
+  nativeBuildInputs = [ pkgs.pkg-config ];
+
+  buildInputs = [ pkgs.xorg.libXrandr pkgs.xorg.libX11 ];
+
+  src = fetchFromGitHub {
+    owner = "neXromancers";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "tJnF1X+NI1hP0J/n3rGy8TD/yIveqRPVlJvJvn0C7Do=";
+  };
+
+  cargoSha256 = "TL2WehcCwygLMVVrBHOX1HgVtDhgVsIgUeiadEjCj1o=";
+
+  meta = with stdenv.lib; {
+    description = "Minimal X screenshot utility";
+    homepage = https://github.com/neXromancers/shotgun;
```

We're discouraging and trying to eventually deprecate the URL syntax. See here for more details: https://github.com/NixOS/rfcs/pull/45

```suggestion
    homepage = "https://github.com/neXromancers/shotgun";
```